bindings/pyroot/pythonizations/src/PyzPythonHelpers.cxx

```diff
@@ -29,6 +28,12 @@ PyROOT extension module.
 
 #include <sstream>
 
+namespace CPyCppyy {
```

Comment:
Can you add a comment that this is needed for Windows?
It should be fine in the other platforms as well?

Reply:
Yes, it's simply a forward declaration